xen/oprofile: Disable callgraph support, it seems to be broken

It seems like oprofile callgraph samples are misaligned and causing corrupt
samples.

diff -r 1a3157247efe xen/include/asm-x86/xenoprof.h
--- a/xen/include/asm-x86/xenoprof.h
+++ b/xen/include/asm-x86/xenoprof.h
@@ -55,7 +55,7 @@ int xenoprofile_get_mode(struct vcpu *, 
 
 static inline int xenoprof_backtrace_supported(void)
 {
-    return 1;
+    return 0;
 }
 
 void xenoprof_backtrace(struct vcpu *, const struct cpu_user_regs *,
